pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -456,7 +456,6 @@ stdenv.mkDerivation {
     "-Dsysvinit-path="
     "-Dsysvrcnd-path="
 
-    "-Dkill-path=${coreutils}/bin/kill"
```

Comment:
Case where we had a non-existent build option. What is hard to know is whether we needed that build option in for compatibility on Nix. i.e. did systemd start hard coding the path to `kill` or did they implement `kill` inside of systemd?

Reply:
[They implemented kill inside systemd.](https://github.com/systemd/systemd/commit/9a85778412fa3e3f8d4561064131ba69f3259b28)

Identified with `git log -pG kill-path` in the systemd repo.